pkg/config/v3/config.go

```diff
@@ -47,12 +47,11 @@ type cfg struct {
 	Resources []resource.Resource `json:"resources,omitempty"`
 
 	// Plugins
-	Plugins PluginConfigs `json:"plugins,omitempty"`
+	Plugins pluginConfigs `json:"plugins,omitempty"`
```

Comment:
Why we cannot export the PluginConfigs here?

Reply:
It doesn't make much sense to export it because is something expecific of v3 implementation of the config file. Previously it was exported because it was needed by the `pkg/cli/internal/config` package that has been replaced here and is no longer needed outside of this package.

Reply:
So, I understand that we will be able to read/update and etc it by what you are calling here as yamlStore and store right?